cloud_foundry_api/assets/service_checks.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Could we have a check to make sure the api works, like a status check?

Reply:
yup